tests/cupy_tests/statics_tests/test_order.py

```diff
@@ -80,18 +84,20 @@ def test_percentile_out(self, xp, dtype, interpolation):
 
     @for_all_interpolations()
     @testing.for_all_dtypes(no_float16=True, no_bool=True, no_complex=True)
-    @testing.numpy_cupy_raises(accept_error=ValueError)
-    def test_percentile_bad_q(self, xp, dtype, interpolation):
-        a = testing.shaped_random((4, 2, 3, 2), xp, dtype)
-        q = testing.shaped_random((1, 2, 3), xp, dtype=dtype, scale=100)
-        return xp.percentile(a, q, axis=-1, interpolation=interpolation)
+    def test_percentile_bad_q(self, dtype, interpolation):
+        for xp in (numpy, cupy):
+            a = testing.shaped_random((4, 2, 3, 2), xp, dtype)
+            q = testing.shaped_random((1, 2, 3), xp, dtype=dtype, scale=100)
+            with pytest.raises(ValueError):
+                xp.percentile(a, q, axis=-1, interpolation=interpolation)
 
     @testing.for_all_dtypes(no_float16=True, no_bool=True, no_complex=True)
-    @testing.numpy_cupy_raises(accept_error=ValueError)
-    def test_percentile_unexpected_interpolation(self, xp, dtype):
-        a = testing.shaped_random((4, 2, 3, 2), xp, dtype)
-        q = testing.shaped_random((5,), xp, dtype=dtype, scale=100)
-        return xp.percentile(a, q, axis=-1, interpolation='deadbeef')
+    def test_percentile_uected_interpolation(self, dtype):
```

Comment:
```suggestion
    def test_percentile_unexpected_interpolation(self, dtype):
```

Reply:
Oh... my keyboard macro automatically removed `xp`s at the line after `numpy_cupy_raises`... 😿

Reply:
Fixed.